.npmignore

```diff
@@ -1,5 +1,7 @@
-examples/
-src/
+examples/*
+!examples/js/
+src/*
+!src/extras/
```

Comment:
Sure, so it first excludes a folder(so that whole of src is not included, for example), and then includes a specific sub-folder inside it to be included in the package(src/extras, for example).
To test it out and use the package while this PR is waiting, I have created an npm package, that uses the exact changes done in this particular PR here, that you can check out to have a look at what exactly results from these rules : https://www.npmjs.com/package/threejs-extras-examples